login_service/sparql_queries.rb

```diff
@@ -42,7 +42,7 @@ def select_account_by_session(session)
       query += "                  <#{MU_EXT.sessionGroup}> ?group ."
       query += "   }"
       query += "   GRAPH <#{graph}> {"
-      query += "     ?group a <#{BESLUIT.Bestuurseenheid}> ;"
+      query += "     ?group a <#{ORG.Organization}> ;"
```

Comment:
If these are the only changes, which would be cool, then I would suggest to move this to an env variable; make the default as bestuurseenheid. And override it in subsidiedatabank.

Reply:
That way we can keep it reverse compatible; we might consider later to introduce a breaking change. but that's a bit of a pain for all apps.

Reply:
don't make it overly complicated BTW; just allow plain URI's in setting up the env variable.

Reply:
GROUP_TYPE env added